src/lib/web3-utils.js

```diff
@@ -5,7 +5,7 @@ import { toChecksumAddress } from 'web3-utils'
 import env from '../environment'
 import { getDefaultChain } from '../local-settings'
 
-const DEFAULT_LOCAL_CHAIN = ''
+const DEFAULT_LOCAL_CHAIN = 'i'
```

Comment:
Not sure, but should this be `1`?

Reply:
Yep! Fixed over in the `last-web-features` branch.